runestone/usageAssignment/__init__.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (C) 2011  Bradley N. Miller
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+__author__ = 'Paul Resnick'
+
+from docutils import nodes
+from docutils.parsers.rst import directives
+from docutils.parsers.rst import Directive
+from sqlalchemy import create_engine, Table, MetaData, select, delete
+from sqlalchemy.orm import sessionmaker
+
+
+def setup(app):
+    app.add_directive('usageassignment',usageAssignment)
```

Comment:
For usage examples see directives near the top of https://raw.githubusercontent.com/presnick/ProgramsInformationPeople/f15/_sources/Assignments/ps5.rst